### ISSUE TITLE
Choose Frontend Framework

### ISSUE THREAD

**Issue by tranquiliza** [tranquiliza/MealRoulette#17]
Currently ASP.NET MVC is the most likely choice, however there are possiblities for Angular or other Frameworks for the web. 

Perhabs another Framework for Crossplatform (Electron? )

**Comment by tranquiliza** [comment]
So far angular seems like an interesting choice

**Comment by tranquiliza** [comment]
Angular has many pros n cons, and after working with it for a while. I am considering ditching Angular in favor for basic HTML serving by the API (MVC) 

 Angular brings many interesting things to the table, but makes using Materialize styling and other javascript frameworks with it, a nightmare. 

I would like to use Materialize because this has some interesting design concepts and is well defined by the documentation delivered by Google. 

Angular on the other hand have many huge learning curves and is a framework that involves a lot of "Work arounds" just to do a simple thing as calling a javascript function on docuement.ready event...

**Comment by tranquiliza** [comment]
Doing this would also solve another potential issue, regarding downtime of the product. 

If the API is down, the Angular app doesn't work. 
So the Angular app is in fact a 1-1 with the API so seperating it, causes no real advantage other than using 2 different frameworks to solve the same issue. 

tldr.: Moving from Angular to .NET MVC instead. 
API and Frontend are combined, API is still there and will be the primary use for access to data. 
This allows for javascript to access the objects needed and if others do not like the frontend, they can in fact design their own, using the documentation provided by the API
